examples/hselect.rs

```diff
@@ -0,0 +1,28 @@
+extern crate session_types;
+
+use session_types::*;
+
+fn main() {
+    let (tcs, rcs) = session_channel();
+    let (tcu, rcu) = session_channel();
+
+    let receivers = vec!(rcs, rcu);
+
+    let () = tcs.send("Hello, World from TCS!".to_string()).close();
```

Comment:
This pattern looks odd to me, `let () = …`. I thought the idiomatic way to discard results is `let _ = …`. In this case, I don't think calls to `.close()` have `#[must_use]`